db/Models/get.models.js

```diff
@@ -4,7 +4,11 @@ const db = require("../connection")
 function fetchTopics (){
     return db.query("SELECT * FROM topics")
     .then((res)=>{
-        return res.rows
+        if(res.rows.length === 0){
+            return Promise.reject({status: 204, message: "No Content"})
+        } else {
+            return res.rows
+        }
```

Comment:
You don’t need to check if there are no rows because PSQL will just return an empty array. Please remove this check as it's not necessary 😄